### DISCUSSION TITLE
Add Digilent  Nexys A7 Artix-7 FPGA

### DISCUSSION THREAD

**Discussion by veso266** [FPGAwars/icestudio#819] [New Boards Requests]
Hi there, not sure how easy is to add another board to this program, normaly you have to use [Vivado](https://www.xilinx.com/products/design-tools/vivado.html)

but I wanted to try Icestudio it seems easy to work with and it better beginner frendly then Vivado is

Its boardfiles are here: https://github.com/Digilent/vivado-boards/archive/master.zip if it helps any

**Comment by AcThPaU** [comment]
The Symbiflow tool chain is completely stable now: https://f4pga.org/
I can't see why it cannot be added to icestudio.

**Comment by veso266** [comment]
Oh thats great
why does it need Symbiflow tool chain, I thought IceStudio uses APIO, but if I saw this APIO does not support  Nexys A7 Artix-7 FPGA

**Comment by cavearr** [comment]
Hi! we are working on it in the next months this feature will be available, maintain this thread open to up to date.

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
